ibis/impala/compiler.py

```diff
@@ -72,18 +74,7 @@ def table_set_formatter(self):
         return ImpalaTableSetFormatter
 
 
-class ImpalaTableSetFormatter(comp.TableSetFormatter):
-
-    _join_names = {
-        ops.InnerJoin: 'INNER JOIN',
-        ops.LeftJoin: 'LEFT OUTER JOIN',
-        ops.RightJoin: 'RIGHT OUTER JOIN',
-        ops.OuterJoin: 'FULL OUTER JOIN',
-        ops.LeftAntiJoin: 'LEFT ANTI JOIN',
-        ops.LeftSemiJoin: 'LEFT SEMI JOIN',
-        ops.CrossJoin: 'CROSS JOIN',
-    }
-
+class ImpalaTableSetFormatter(BaseTableSetFormatter):
     def _get_join_type(self, op):
```

Comment:
Im not sure but i think by switching from Impala (where the below is done to make CrossJoin) to Base where it becomes Inner is causing the test failure we had.  I updated the test to expect `INNER JOIN` instead of `CROSS JOIN` just to see if it would work.  If it does work i guess that would make this an impala specific test that would need to be moved instead of updating the `expected` result.  I saw some other `CROSS JOIN` tests though and i dont think they were failing.  Maybe its only when with no predicates per this function.  I need to dig into it more.

Reply:
Looks like its working.

I was thinking of leaving this test with the updated expected result to still cover what the test was designed for and then also adding it to impala for handling the translation to `CROSS JOIN`.  What do you think?